From 9ad3608e35732a3b9df3327f8f5f31c551303552 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 20 Jul 2020 16:28:23 +0800
Subject: [PATCH 1/1] Test case for changing .h file included in the driver .asm file

Description: Change the define value of 'CPSR_MODE_ABORT' in the ArmPkg/Include/Chipset/ArmV7.h

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 ArmPkg/Include/Chipset/ArmV7.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ArmPkg/Include/Chipset/ArmV7.h b/ArmPkg/Include/Chipset/ArmV7.h
index fe91031ef33d..8a8b6c580231 100644
--- a/ArmPkg/Include/Chipset/ArmV7.h
+++ b/ArmPkg/Include/Chipset/ArmV7.h
@@ -32,7 +32,7 @@
 #define CPSR_MODE_FIQ        0x11
 #define CPSR_MODE_IRQ        0x12
 #define CPSR_MODE_SVC        0x13
-#define CPSR_MODE_ABORT      0x17
+#define CPSR_MODE_ABORT      0x14
 #define CPSR_MODE_HYP        0x1A
 #define CPSR_MODE_UNDEFINED  0x1B
 #define CPSR_MODE_SYSTEM     0x1F
-- 
2.27.0.windows.1

